Common/cpp/Registries/EventHandlerRegistry.cpp

```diff
@@ -36,6 +37,13 @@ void EventHandlerRegistry::processEvent(
         handlersForEvent.push_back(handler.second);
       }
     }
+    auto eventHash = std::to_string(emitterReactTag) + eventName;
```

Comment:
I'm wondering if we can use `std::make_pair(emitterReactTag, eventName)` as hash type instead of concatenation.

Reply:
Great Idea! As we discussed offline, I'll do it in another PR.